javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,14 @@
+var fullName = 'Colin Macrae';
+var isCool = false;
+var totalPets = 0;
+console.log('Full name:', fullName);
+console.log('Is cool?:', isCool);
+console.log('Number of pets:', totalPets);
+
```

Comment:
You should also have console logs for `typeof fullName` `typeof isCool` & `typeof totalPets`